requirements.txt

```diff
@@ -1,10 +1,10 @@
 pandas>=0.15
 seaborn>=0.7.0
 scipy>=0.17.0
-topiary>=0.0.20, <0.1.0
```

Comment:
We talked offline about using these ranges going forward. (Just adding context.)